proof/refine/ARM/KHeap_R.thy

```diff
@@ -917,7 +960,7 @@ lemma setObject_other_corres:
   apply (clarsimp simp add: caps_of_state_after_update cte_wp_at_after_update
                             swp_def fun_upd_def obj_at_def)
   apply (subst conj_assoc[symmetric])
-  apply (rule conjI[rotated])
+  apply (extract_conjunct \<open>match conclusion in "ghost_relation _ _ _" \<Rightarrow> -\<close>)
```

Comment:
Thanks for making more use of `extract_conjunct` throughout this work, it's definitely more maintainable than the bare `rule conjI`s